### ISSUE TITLE
producer not stop after shutdown since #3199

### ISSUE THREAD

**Issue by yuz10** [apache/rocketmq#3609]
**BUG REPORT**

1. Please describe the issue you observed:

- What did you do (The steps to reproduce)?
```
   public static void main(String[] args) throws MQClientException, InterruptedException {

        DefaultMQProducer producer = new DefaultMQProducer("ProducerGroupName");
        producer.setNamesrvAddr("localhost:9876");
        producer.start();
        producer.shutdown();
    }
```
- What did you expect to see?
   the producer will exit.

- What did you see instead?
  the producer dont exit forever.

2. Please tell us about your environment:

3. Other information (e.g. detailed explanation, logs, related issues, suggestions how to fix, etc):

**Comment by panzhi33** [comment]
I was fixed，can see #3453

**Comment by yuz10** [comment]
> I was fixed，can see #3453

See, I will close this.
